editor/scene/sprite_frames_editor_plugin.cpp

```diff
@@ -544,7 +544,11 @@ bool SpriteFramesEditor::_matches_background_color(const Color &p_background_col
 
 Size2i SpriteFramesEditor::_estimate_sprite_sheet_size(const Ref<Texture2D> p_texture) {
 	Ref<Image> image = p_texture->get_image();
-	Size2i size = p_texture->get_size();
+	if (image->is_compressed()) {
+		image = image->duplicate();
```

Comment:
Does this have some potential for a performance regression? Is the function called often? Might be worth commenting, or opening a follow-up PR for if so.